akka-persistence-typed/src/main/scala/akka/persistence/typed/javadsl/ReplicatedEventSourcing.scala

```diff
@@ -122,4 +122,22 @@ object ReplicatedEventSourcing {
     eventSourcedBehaviorFactory(context)
   }
 
+  /**
+   * Initialize a replicated event sourced behavior.
+   *
+   * Events from each replica for the same entityId need to be passed to it by an external stream.
+   * Care must be taken to handle events in any order as events can happen concurrently at different replicas.
+   *
+   * Using a replicated event sourced behavior means there is no longer the single writer guarantee.
+   *
+   * The journal plugin id for the entity itself can be configured using withJournalPluginId after creation.
+   */
+  def externalReplication[Command, Event, State](replicationId: ReplicationId, allReplicas: JSet[ReplicaId])(
```

Comment:
`@ApiMayChange` until we are more sure about the final api?

Reply:
Maybe I should just make it akka-internal for now, the way to actually publish events is anyway not public API.

Reply:
sounds good `@InternalStableApi` (I know you don't like it, but public api come more drawbacks)

Reply:
That also probably means we can skip the Java API, so less changes here. 👍